src/test/java/array/Array2dTest.java

```diff
@@ -98,17 +103,14 @@ void shouldVisitNonEmptyArray() {
                 .forEach(i -> array.insert(
                         Coordinates2d.of(i, i),
                         10 * i));
-        String expectedPresentation = "0, null, null, null, null, \n" +
-                                      "null, 1, null, null, null, \n" +
-                                      "null, null, 2, null, null, \n" +
-                                      "null, null, null, 3, null, \n" +
-                                      "null, null, null, null, null, \n";
+        String expectedPresentation = "0, null, null, null, null, " +
```

Comment:
For me, it's a bad presentation of 2D Array. It seems like 1D. Then let's switch to `toString` method that will be know all the details of realization (2D vs 1D)

Reply:
Well, we could do smth like this if you like insist

```
String expectedPresentation = "0, null, null, null, null, \n" +
                                      "null, 10, null, null, null, \n" +
                                      "null, null, 20, null, null, \n" +
                                      "null, null, null, 30, null, \n" +
                                      "null, null, null, null, null, \n";
        StringBuilder sb = new StringBuilder();
        // when
        int counter[] = { 1 };
        array.visit(cell -> {
            sb.append(cell).append(", ");
            if (counter[0] % size == 0) {
                sb.append("\n");
            }
            counter[0]++;
        });
```

but I don't see any problem in a way internal structure is presented in tests now.

And using `toString()` to get this representation IMHO is using `toString()` not in a way it was designed to.

Reply:
ok